apps/web/components/booking/BookingListItem.tsx

```diff
@@ -353,6 +353,28 @@ function BookingListItem(booking: BookingItemProps) {
                   attendees={booking.attendees}
                 />
               </div>
+              {booking.location && booking.location.startsWith("https://") && (
+                <Link href={booking.location} className="text-sm leading-6 text-blue-400 hover:underline">
+                  {booking.location.startsWith("https://zoom") ? (
+                    <div className="flex items-center gap-1">
+                      <img src="/app-store/zoomvideo/icon.svg" alt="Zoom Logo" height={16} width={16} />
+                      Join Zoom
+                    </div>
+                  ) : booking.location.startsWith("https://meet.google") ? (
+                    <div className="flex items-center gap-1">
+                      <img
+                        src="/app-store/googlevideo/logo.webp"
+                        alt="Google Meet Logo"
+                        height={16}
+                        width={16}
+                      />
+                      Join Google Meet
+                    </div>
+                  ) : (
+                    "Meeting Link"
+                  )}
+                </Link>
+              )}
```

Comment:
we should not hard code logos and names, instead use whatever conference location was used